### ISSUE TITLE
Creation of leaderboard

### ISSUE THREAD

**Issue by AditiKhandelia** [CS455-Assignment-1/StackNServe#35]
```
As a developer,
I want the app to have a leaderboard
so that users can track their progress, compare their performance with others, and stay motivated to improve their ranking over time.
```

**Comment by AditiKhandelia** [comment]
The leaderboard interface design can be accessed at https://www.canva.com/design/DAGNPAtcjYQ/WI_57OYJ3qqM8Mc39acChw/edit?utm_content=DAGNPAtcjYQ&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

**Comment by AditiKhandelia** [comment]
Closed with 0309525e843604b751c3cc5856fe022934a641bd
